packages/user-interface/src/pages/task/task-page.tsx

```diff
@@ -30,7 +33,7 @@ const TaskPage = () => {
     variables: {id: formId},
   });
 
-  const {data: taskData} = useGetTaakByIdQuery({variables: {id: taskId}});
+  const {data: taskData, error} = useGetTaakByIdQuery({variables: {id: taskId}});
```

Comment:
Kunnen we er hier vanuit gaan dat een error altijd "unAuthorized" betekend?

Reply:
Tja daar zat ik een beetje mee. Als de taak niet gevonden mag worden dan krijg je een empty list error terug, zou natuurlijk ook dat de api niet gevonden kan worden door netwerk issue.
Standaard error tekst was een algemene foutmelding. Dat leek mij ook niet correct

Reply:
Kunnen we in de graph api ook later response codes toevoegen? Dan kunnen we die beter verwerken in de FE. Voor nu misschien een algemene melding.

Reply:
ik zal het algemene foutmelding maken. En ik zou moeten uitzoeken hoe we een response code kunnen toevoegen aan de response